src/mutations/content/translations/useCreateContentTypeTranslation.ts

```diff
@@ -0,0 +1,72 @@
+import { GetAdminAPI } from "@src/AdminAPI";
+import { ConnectedXMResponse, ContentTypeTranslation } from "@src/interfaces";
+import {
+  MutationOptions,
+  MutationParams,
+  useConnectedMutation,
+} from "@src/mutations/useConnectedMutation";
+
+/**
+ * @category Params
+ * @group Content-Translation
+ */
+export interface CreateContentTypeTranslationParams extends MutationParams {
+  contentTypeId: string;
+  locale: string;
+  autoTranslate?: boolean;
+}
+
+/**
+ * @category Methods
+ * @group Content-Translation
+ */
+export const CreateContentTypeTranslation = async ({
+  contentTypeId,
+  locale,
+  autoTranslate,
+  adminApiParams,
+  queryClient,
+}: CreateContentTypeTranslationParams): Promise<
+  ConnectedXMResponse<ContentTypeTranslation>
+> => {
+  const connectedXM = await GetAdminAPI(adminApiParams);
+
+  const { data } = await connectedXM.post(
+    `/contentTypes/${contentTypeId}/translations`,
+    {
+      locale,
+      autoTranslate,
+    }
+  );
+  if (queryClient && data.status === "ok") {
+    queryClient.invalidateQueries({
+      queryKey: CONTENT_TYPE_TRANSLATIONS_QUERY_KEY(contentTypeId),
+    });
+    SET_CONTENT_TYPE_TRANSLATION_QUERY_DATA(
+      queryClient,
+      [contentTypeId, data.locale],
+      data
+    );
+  }
```

Comment:
@RobertUpchurch @joshuackeller wanted you guys' input.
I'm noticing we are comparing data.status === 'ok'. This implies the backend returns an "ok" string inside the actual response for each route. Is this correct? shouldn't we be using the status param from the actual connectedXM.post call?

```suggestion
  const { data, status } = await connectedXM.post(
    `/contentTypes/${contentTypeId}/translations`,
    {
      locale,
      autoTranslate,
    }
  );
  if (queryClient && status === "something") { //not sure how status is populated by TSQ
    queryClient.invalidateQueries({
      queryKey: CONTENT_TYPE_TRANSLATIONS_QUERY_KEY(contentTypeId),
    });
    SET_CONTENT_TYPE_TRANSLATION_QUERY_DATA(
      queryClient,
      [contentTypeId, data.locale],
      data
    );
  }
```

Reply:
This is valid for all other calls, we are using this same comparison everywhere